src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -771,6 +877,17 @@ private static File findJenkinsWar() throws Exception {
         return WarExploder.findJenkinsWar();
     }
 
+    /**
+     * Create a client configured to trust the self-signed certificate used by this instance.
```

Comment:
```suggestion
     * Create a client configured to trust any self-signed certificate used by this instance.
```
(since it also works when HTTPS is not enabled)